### DISCUSSION TITLE
I was being rejected after TONS OF REQUESTS

### DISCUSSION THREAD

**Discussion by MisakaRehana** [community/community#138566] [GitHub Education] [answered]
### Hi everyone,

I was being rejected after tons of requests -- yes, NONE OF THEM was approved.
I tried to clear cookies, disable and re-enable browser location settings, took tons of the CHSI academic online proof pictures (English version -- I even paid for it), set biliing info to my real info, set the name to my full name before application, logout and re-login my account, disable any VPNs I use, capture the CHSI online proof horizontally, vertically, scaled, unscaled -- **ALWAYS RESULTS IN A 'REJECTED' INFORMATION**.

I have no word to say. Maybe GitHub verification system is ??? me by person.

I just wonder if there is a usable solution for this.

**Comment by MisakaRehana** [comment, marked as answer]
I tried to use mobile phone instead of PC to take photos and send application, and the problem solved.

**Comment by aivo9** [comment]
Captain here!

I had the same issue, but I just got approved.

Got rejected multiple times. Time after time. Made few perceptions about how it works, and finally succeeded!

1. Put everything in order in your account settings. Just as the instructions advice you to do. Name, emails, bios, 2 phase authentication, etc.
2. Try to photograph different kind of official school documents with a bad webcam and get rejected multiple times over and over again.
3. Get frustrated.
4. Realize that there is no human approving your entries but artificial intelligence instead "reading" words from the pictures you send.
5. Improvise, adapt, overcome.
6. Open the notebook of your mobile phone and type:

[Your University of Fancy Science]

Student: [insert your name here]
School: [insert your school name (same as in the title) here]
Education: [insert the name of your education here] (I had to add ",Virtual learning" to it cuz I'm not on campus right now, and it got mad)
Study right: [insert the day when you got your study right and the day it ends (in US format e.g. 1 January 2024 - 31 December 2028)]

7. So basically everything is true and you are not lying, but you just made the proof yourself on your mobile phone's notepad.
8. Take a screenshot. Edit the picture with your mobile phone and cut off all the excess space so that the text looks as big as possible in the screen.
9. Open the edited picture on your mobile phone and take the phone so close to your laptop's webcam so that all the words are clearly visible in the picture.
10. Take the photo and check that all the information is visible in the photo.
11. Submit your application.
12. Get approved few minutes later.
13. Not your fault the system of approval is so horrible and broken so feel no shame about cutting the corners.
14. Return the favor and go help others.
15. Enjoy!

Welcome to the community!

*flies away*

**Comment by devv-is-coding** [comment]
I have a similar problem. I had tons of rejections about applying to Copilot Pro as a student learner, but I do understand it since my front ID has no valid date on it. My ID is back-to-back, where the front is my information, and the back has the validity of my ID and the contact information of my guardian in case of emergency, I had to take both the front and back of my ID a picture but cannot send both. I don't understand why they have limited to one photo per submission and the validation of the system sucks. Please help me with this.

**Comment by hassanpatroliumsarvice-stack** [comment]
[](url) @aivo9
